src/data/accessibility-quiz.ts

```diff
@@ -1,4 +1,159 @@
 const accessibilityQuiz = [
+  {
+    Question: "What role does the 'alt' attribute play when adding alternative text to images on a webpage?",
+    Answer: 
+      "The 'alt' attribute describes the content of the image to users who cannot see it.",
+    Distractor1: "The 'alt' attribute enhances image aesthetics.",
+    Distractor2: "The 'alt' attribute speeds up image loading.",
+    Distractor3: "The 'alt' attribute provides translations for the image content.",
+    Explanation: 
+      "The 'alt' attribute in alternative text describes image content to users who cannot see the image.",
+    Link: "https://www.freecodecamp.org/news/common-accessibility-errors-and-how-to-fix-them/"
+  },
+  {
+    Question: "What is the purpose of the 'Chrome Accessibility Developer Tools' extension for Google Chrome?",
+    Answer: 
+    "This extension adds an Accessibility sidebar pane to the Chrome Developer tools and can be used to check accessibility properties of web elements.",
+    Distractor1: "It's a new JavaScript framework",
+    Distractor2: "It's a sorting algorithm",
+    Distractor3: "It's a type of programming paradigm",
+    Explanation: 
+    "The extension aids in inspecting and ensuring web accessibility by providing relevant information in the Developer tools.",
+    Link: "https://www.freecodecamp.org/news/accessibility-testing-tools-for-ada-compliance/"
+  },
+  {
+    Question: "Which tool or extension allows you to find accessibility defects on your web page in both Chrome and Firefox?",
+    Answer: 
+    "WAVE",
+    Distractor1: "Color Oracle",
+    Distractor2: "aXe",
+    Distractor3: "ChromeVox",
+    Explanation: 
+    "WAVE is an extension available for both Chrome and Firefox that helps identify accessibility issues on web pages.",
+    Link: "https://www.freecodecamp.org/news/accessibility-testing-tools-for-ada-compliance/"
+  },
+  {
+    Question: "What is the primary function of a screen reader in web development?",
+    Answer: 
+    "Screen readers are used to convey the content of a webpage to users with visual impairments via text-to-speech, helping developers understand and create accessible web content.",
+    Distractor1: "It's a web design tool",
+    Distractor2: "It's a code editor",
+    Distractor3: "It's a graphics rendering software",
+    Explanation: 
+    "Screen readers assist users with visual impairments in accessing web content through spoken text.",
+    Link: "https://www.freecodecamp.org/news/accessibility-testing-tools-for-ada-compliance/"
+  },
```

Comment:
we already have a similar question in the file.
please remove this

Reply:
okay